community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -101,11 +101,11 @@ def instance_properties(nodeset:object, model:str, placement_group:Optional[str]
         props.resourcePolicies = [placement_group]
 
     if reservation := lookup().nodeset_reservation(nodeset):
-        update_reservation_props(reservation, props, placement_group)
+        update_reservation_props(reservation, props, placement_group, False)
```

Comment:
please use it as named arg `calendar_res=False`
also, consider renaming it to `calendar_mode`